agent/api/task/task.go

```diff
@@ -1747,7 +1747,6 @@ func (task *Task) dockerConfig(container *apicontainer.Container, apiVersion doc
 		Env:          dockerEnv,
 	}
 
-	// TODO [SC] - Move this as well as 'dockerExposedPorts' SC-specific logic into a separate file
```

Comment:
Q: Just curious, any reason why we're removing this TODO comment here? Not sure if I have the full context for this.

Reply:
This was unintentional. I will add that back in.